src/components/deck-gl-wrapper/mesh-layer/mesh-layer-fragment.glsl.ts

```diff
@@ -0,0 +1,55 @@
+export default `#version 300 es
```

Comment:
comment changed code. If you removed something, keep removed code commented.
If no changes, leave a comment above that the file has been moved from `deck.gl` without changes to handle the fork of Tile3DLayer

Reply:
No changes from my side, added appropriate comment